### PR TITLE
dec dev update

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: content/posts/irods-development-update-december-2023.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+
+Support for `podman` was added to the iRODS Testing Environment. Big thanks to Marco Roeland for the contribution.
+
+Some users may have noticed we haven't released the Indexing Capability plugin for iRODS 4.3.1. The reason for this is captured in [issue #128](https://github.com/irods/irods_capability_indexing/issues/128). TLDR: We have to replace functionality provided by the elasticlient/CPR library. If you have suggestions on how to best handle this, please leave a comment in the issue.
+
+I think it's safe to say iRODS has had a fantastic year. iRODS 4.2.12 closes out the 4.2 series with improved stability and a set of features that allow users to get work done. The release of iRODS 4.3.1 is a huge accomplishment. It is the definitive version of iRODS and sets the stage for increasing adoption. We also want to thank the community for its support and we ask that you continue to open issues and think of interesting ways to improve data management for everyone.
+
+Happy Holidays and see you in new year!
```
Comment (trel) [inline content/posts/irods-development-update-december-2023.md:26]: the new year

File: content/posts/irods-development-update-december-2023.md [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+            - implemented HeadBucket
+        - removing Boost.URL submodule, will use externals package instead
+        - will be refactored to not use coroutines, to help the transition to libstdc++
+
+    - [Jargon](https://github.com/DICE-UNC/jargon)
+        - preparing release of 4.3.3.0
+        - added option to test framework that allows users to install other versions of iRODS 4.3
+        - updated FilePermissionEnum to support new permission levels
+        - added server version detection methods for iRODS 4.3
+        - StartupPack now sends the correct value for <option> element
```
Comment (trel) [inline content/posts/irods-development-update-december-2023.md:167]: the option brackets don't render as you want - look at &lt and &gt.

File: content/posts/irods-development-update-december-2023.md [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+
+    - [NFSRODS](https://github.com/irods/irods_client_nfsrods)
+        - preparing 2.2.0 release
+        - changed base OS of Docker image to Ubuntu 22.04
+        - server clears more caches to avoid stale file handle errors
+        - removed unnecessary stat in list operation
+        - investigating performance improvements related to concurrent usage
+
+    - mungefs
```
Comment (trel) [inline content/posts/irods-development-update-december-2023.md:176]: link here as well?

**Review by trel — COMMENTED** [review]
just some tweaks.

File: content/posts/irods-development-update-december-2023.md [hunk 4]
```diff
@@ -0,0 +1,274 @@
+Title: iRODS Development Update: December 2023
+Date: 2023-12-15 19:00
```
[the remark this answers is not in the document]
Comment (korydraughn) [reply content/posts/irods-development-update-december-2023.md]: done

File: content/posts/irods-development-update-december-2023.md (hunk 1 above)
Comment (korydraughn) [reply content/posts/irods-development-update-december-2023.md]: done

File: content/posts/irods-development-update-december-2023.md (hunk 2 above)
Comment (korydraughn) [reply content/posts/irods-development-update-december-2023.md]: done

File: content/posts/irods-development-update-december-2023.md (hunk 3 above)
Comment (korydraughn) [reply content/posts/irods-development-update-december-2023.md]: done

**Review by trel — COMMENTED** [review]
yep, squash it.   we can merge in the morning.

**Comment by korydraughn** [comment]
I've squashed the commits, but let me confirm that the statement about PRC/PAM is true before merging.

**Comment by korydraughn** [comment]
We're all set. Merge when ready.

**Pull request merged**